Tests/HtmlTests/AttributesTests.swift

```diff
@@ -108,6 +108,11 @@ final class AttributesTests: XCTestCase {
       """,
       render(.input(attributes: [.autofocus(false), .checked(false), .disabled(false), .readonly(false), .required(false)]))
     )
+    XCTAssertEqual(
+      """
+			<input type="file" required accept=".pdf, image/*">
```

Comment:
This seems to be using tab stops instead of spaces. Could you reformat to be 2 space tabs?

Reply:
Yeah, sorry about that. Should be fixed in the latest version 👍